sail-riscv not compiling

Did a fresh clone of the repository and started building the simulator, but ran into the following error.
```
❯ env DOWNLOAD_GMP=FALSE ./build_simulators.sh
-- The C compiler identification is GNU 15.1.1
-- The CXX compiler identification is GNU 15.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test LINKER_SUPPORTS_NO_UNDEFINED
-- Performing Test LINKER_SUPPORTS_NO_UNDEFINED - Success
-- Performing Test HAVE_WSELF_ASSIGN
-- Performing Test HAVE_WSELF_ASSIGN - Failed
-- Performing Test HAVE_WIMPLICIT_FALLTHROUGH
-- Performing Test HAVE_WIMPLICIT_FALLTHROUGH - Success
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.4.3")
-- Found GMP: /usr/lib/libgmp.so
-- Found sail: $HOME/.opam/default/bin/sail
-- Sail library directory: $HOME/.opam/default/share/sail
-- Configuring done (0.5s)
-- Generating done (0.2s)
-- Build files have been written to: $HOME/sail-riscv/build
[  0%] Building C code from Sail model (rv32d)
[  1%] Building C object sail_runtime/CMakeFiles/sail_runtime.dir/$HOME/.opam/default/share/sail/lib/rts.c.o
[  1%] Built target default_config
[  1%] Building C object sail_runtime/CMakeFiles/sail_runtime.dir/$HOME/.opam/default/share/sail/lib/elf.c.o
[  1%] Building C code from Sail model (rv64d)
[  1%] Building C object sail_runtime/CMakeFiles/sail_runtime.dir/$HOME/.opam/default/share/sail/lib/sail_failure.c.o
[  1%] Building C object sail_runtime/CMakeFiles/sail_runtime.dir/$HOME/.opam/default/share/sail/lib/sail.c.o
[  1%] Building C object sail_runtime/CMakeFiles/sail_runtime.dir/$HOME/.opam/default/share/sail/lib/cJSON.c.o
[  2%] Building C object sail_runtime/CMakeFiles/sail_runtime.dir/$HOME/.opam/default/share/sail/lib/sail_config.c.o
[  2%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_f32UIToCommonNaN.c.o
[  2%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_commonNaNToF32UI.c.o
[  2%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_commonNaNToF64UI.c.o
[  3%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_f64UIToCommonNaN.c.o
[  3%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_propagateNaNF128UI.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_propagateNaNF64UI.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_propagateNaNF32UI.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/s_propagateNaNF16UI.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/RISCV/softfloat_raiseFlags.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_add.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_div.c.o
[  5%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_eq.c.o
[  6%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_eq_signaling.c.o
[  6%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_le.c.o
[  6%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_lt.c.o
[  6%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_le_quiet.c.o
[  7%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_lt_quiet.c.o
[  7%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_mul.c.o
[  7%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_rem.c.o
[  7%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_roundToInt.c.o
[  9%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_sqrt.c.o
[  9%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_sub.c.o
[  9%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_f128M.c.o
[  9%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_f16.c.o
[  9%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_f32.c.o
[ 10%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_f64.c.o
[ 10%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_i32_r_minMag.c.o
[ 10%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_i32.c.o
[ 10%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_i64.c.o
[ 10%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_i64_r_minMag.c.o
[ 11%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_ui32.c.o
[ 11%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_ui32_r_minMag.c.o
[ 11%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_ui64.c.o
[ 11%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80M_to_ui64_r_minMag.c.o
[ 13%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_div.c.o
[ 13%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_add.c.o
[ 13%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_eq.c.o
[ 13%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_eq_signaling.c.o
[ 13%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_isSignalingNaN.c.o
[ 14%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_le.c.o
[ 14%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_le_quiet.c.o
[ 14%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_lt.c.o
[ 14%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_lt_quiet.c.o
[ 15%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_mul.c.o
[ 15%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_roundToInt.c.o
[ 15%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_rem.c.o
[ 15%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_sqrt.c.o
[ 15%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_sub.c.o
[ 17%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_f128.c.o
[ 17%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_f16.c.o
[ 17%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_f32.c.o
[ 17%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_f64.c.o
[ 18%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_i32.c.o
[ 18%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_i32_r_minMag.c.o
[ 18%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_i64.c.o
[ 18%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_i64_r_minMag.c.o
[ 18%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_ui32.c.o
[ 19%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_ui32_r_minMag.c.o
[ 19%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_ui64.c.o
[ 19%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/extF80_to_ui64_r_minMag.c.o
[ 19%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_add.c.o
[ 21%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_div.c.o
[ 21%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_eq.c.o
[ 21%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_eq_signaling.c.o
[ 21%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_le.c.o
[ 21%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_le_quiet.c.o
[ 22%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_lt.c.o
[ 22%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_lt_quiet.c.o
[ 22%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_mul.c.o
[ 22%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_mulAdd.c.o
[ 23%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_rem.c.o
[ 23%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_roundToInt.c.o
[ 23%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_sqrt.c.o
[ 23%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_sub.c.o
[ 25%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_extF80M.c.o
[ 25%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_f16.c.o
[ 25%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_f32.c.o
[ 25%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_f64.c.o
[ 26%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_i32.c.o
[ 26%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_i32_r_minMag.c.o
[ 26%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_i64.c.o
[ 26%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_ui32_r_minMag.c.o
[ 26%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_ui32.c.o
[ 26%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_i64_r_minMag.c.o
[ 27%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_ui64_r_minMag.c.o
[ 27%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128M_to_ui64.c.o
[ 27%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_add.c.o
[ 27%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_div.c.o
[ 28%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_eq.c.o
[ 28%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_isSignalingNaN.c.o
[ 28%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_eq_signaling.c.o
[ 28%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_le.c.o
[ 28%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_le_quiet.c.o
[ 30%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_lt.c.o
[ 30%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_lt_quiet.c.o
[ 30%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_mul.c.o
[ 30%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_mulAdd.c.o
[ 30%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_rem.c.o
[ 31%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_sqrt.c.o
[ 31%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_roundToInt.c.o
[ 31%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_sub.c.o
[ 31%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_extF80.c.o
[ 31%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_f32.c.o
[ 32%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_f16.c.o
[ 32%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_f64.c.o
[ 32%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_i32.c.o
[ 32%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_i32_r_minMag.c.o
[ 34%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_i64.c.o
[ 34%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_i64_r_minMag.c.o
[ 34%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_ui32.c.o
[ 34%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_ui32_r_minMag.c.o
[ 35%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_ui64.c.o
[ 35%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f128_to_ui64_r_minMag.c.o
[ 35%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_add.c.o
[ 35%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_div.c.o
[ 35%] Linking C static library libsail_runtime.a
[ 35%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_eq.c.o
[ 36%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_eq_signaling.c.o
[ 36%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_isSignalingNaN.c.o
[ 36%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_le.c.o
[ 36%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_le_quiet.c.o
[ 38%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_lt.c.o
[ 38%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_lt_quiet.c.o
[ 38%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_mul.c.o
[ 38%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_mulAdd.c.o
[ 38%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_rem.c.o
[ 39%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_roundToInt.c.o
[ 39%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_sub.c.o
[ 39%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_sqrt.c.o
[ 40%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_extF80M.c.o
[ 40%] Built target sail_runtime
[ 40%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_extF80.c.o
[ 40%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_f128.c.o
[ 40%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_f128M.c.o
[ 40%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_f32.c.o
[ 40%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_f64.c.o
[ 42%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_i32.c.o
[ 42%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_i32_r_minMag.c.o
[ 42%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_i64.c.o
[ 42%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_i64_r_minMag.c.o
[ 42%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_ui32_r_minMag.c.o
[ 43%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_ui32.c.o
[ 43%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_ui64.c.o
[ 43%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f16_to_ui64_r_minMag.c.o
[ 44%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_add.c.o
[ 44%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_div.c.o
[ 44%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_eq_signaling.c.o
[ 44%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_eq.c.o
[ 44%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_isSignalingNaN.c.o
[ 46%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_le.c.o
[ 46%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_le_quiet.c.o
[ 46%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_lt_quiet.c.o
[ 46%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_lt.c.o
[ 46%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_mul.c.o
[ 47%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_mulAdd.c.o
[ 47%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_rem.c.o
[ 47%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_roundToInt.c.o
[ 47%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_sqrt.c.o
[ 48%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_sub.c.o
[ 48%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_extF80.c.o
[ 48%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_extF80M.c.o
[ 48%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_f128.c.o
[ 48%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_f128M.c.o
[ 50%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_f16.c.o
[ 50%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_f64.c.o
[ 50%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_i32.c.o
[ 51%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_i32_r_minMag.c.o
[ 51%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_i64.c.o
[ 51%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_i64_r_minMag.c.o
[ 51%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_ui32.c.o
[ 51%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_ui32_r_minMag.c.o
[ 51%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_ui64.c.o
[ 52%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f32_to_ui64_r_minMag.c.o
[ 52%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_add.c.o
[ 52%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_div.c.o
[ 52%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_eq.c.o
[ 53%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_eq_signaling.c.o
[ 53%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_isSignalingNaN.c.o
[ 53%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_le.c.o
[ 53%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_lt.c.o
[ 53%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_le_quiet.c.o
[ 55%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_lt_quiet.c.o
[ 55%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_mulAdd.c.o
[ 55%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_mul.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_roundToInt.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_sqrt.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_rem.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_sub.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_extF80.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_extF80M.c.o
[ 56%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_f128M.c.o
[ 57%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_f128.c.o
[ 57%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_f16.c.o
[ 57%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_f32.c.o
[ 59%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_i32.c.o
[ 59%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_i32_r_minMag.c.o
[ 59%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_i64.c.o
[ 59%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_i64_r_minMag.c.o
[ 60%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_ui32.c.o
[ 60%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_ui32_r_minMag.c.o
[ 60%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_ui64.c.o
[ 60%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/f64_to_ui64_r_minMag.c.o
[ 60%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_extF80.c.o
[ 61%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_extF80M.c.o
[ 61%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_f128.c.o
[ 61%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_f16.c.o
[ 61%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_f128M.c.o
[ 61%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_f32.c.o
[ 63%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i32_to_f64.c.o
[ 63%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_extF80.c.o
[ 63%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_f128.c.o
[ 63%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_extF80M.c.o
[ 63%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_f128M.c.o
[ 64%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_f16.c.o
[ 64%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_f32.c.o
[ 64%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/i64_to_f64.c.o
[ 64%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_add128.c.o
[ 65%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_add256M.c.o
[ 65%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addCarryM.c.o
[ 65%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addComplCarryM.c.o
[ 65%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addM.c.o
[ 65%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addMagsExtF80.c.o
[ 67%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addMagsF128.c.o
[ 67%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addMagsF32.c.o
[ 67%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addMagsF16.c.o
[ 68%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_approxRecip32_1.c.o
[ 68%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_addMagsF64.c.o
[ 68%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_approxRecipSqrt_1Ks.c.o
[ 68%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_approxRecipSqrt32_1.c.o
[ 68%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_approxRecip_1Ks.c.o
[ 68%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_compare128M.c.o
[ 69%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_compare96M.c.o
[ 69%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_countLeadingZeros16.c.o
[ 69%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_countLeadingZeros32.c.o
[ 69%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_countLeadingZeros64.c.o
[ 71%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_countLeadingZeros8.c.o
[ 71%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_eq128.c.o
[ 71%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_invalidExtF80M.c.o
[ 71%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_isNaNF128M.c.o
[ 71%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_le128.c.o
[ 71%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mul128By32.c.o
[ 72%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_lt128.c.o
[ 72%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mul128MTo256M.c.o
[ 72%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mul128To256M.c.o
[ 73%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mul64ByShifted32To128.c.o
[ 73%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mul64To128.c.o
[ 73%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mul64To128M.c.o
[ 73%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mulAddF128.c.o
[ 75%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mulAddF16.c.o
[ 75%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mulAddF32.c.o
[ 75%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_mulAddF64.c.o
[ 75%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_negXM.c.o
[ 75%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normExtF80SigM.c.o
[ 76%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normRoundPackToExtF80.c.o
[ 76%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normRoundPackToF128.c.o
[ 76%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normRoundPackToF16.c.o
[ 76%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normRoundPackToF32.c.o
[ 76%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normRoundPackToF64.c.o
[ 77%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normSubnormalExtF80Sig.c.o
[ 77%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normSubnormalF128Sig.c.o
[ 77%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normSubnormalF16Sig.c.o
[ 77%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normSubnormalF32Sig.c.o
[ 78%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_normSubnormalF64Sig.c.o
[ 78%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_remStepMBy32.c.o
[ 78%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundMToI64.c.o
[ 78%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundMToUI64.c.o
[ 78%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundPackToExtF80.c.o
[ 80%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundPackToF128.c.o
[ 80%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundPackToF16.c.o
[ 80%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundPackToF32.c.o
[ 81%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundToI32.c.o
[ 81%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundPackToF64.c.o
[ 81%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundToI64.c.o
[ 81%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundToUI32.c.o
[ 81%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_roundToUI64.c.o
[ 81%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shiftRightJam128.c.o
[ 82%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shiftRightJam128Extra.c.o
[ 82%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shiftRightJam32.c.o
[ 82%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shiftRightJam256M.c.o
[ 82%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shiftRightJam64.c.o
[ 84%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shiftRightJam64Extra.c.o
[ 84%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftLeft128.c.o
[ 84%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftLeft64To96M.c.o
[ 84%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftLeftM.c.o
[ 84%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRight128.c.o
[ 85%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightExtendM.c.o
[ 85%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightJam128.c.o
[ 85%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightJam128Extra.c.o
[ 85%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightJam64.c.o
[ 86%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightJam64Extra.c.o
[ 86%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightJamM.c.o
[ 86%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_shortShiftRightM.c.o
[ 86%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_sub128.c.o
[ 86%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_sub1XM.c.o
[ 88%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_sub256M.c.o
[ 88%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_subM.c.o
[ 88%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_subMagsExtF80.c.o
[ 88%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_subMagsF128.c.o
[ 89%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_subMagsF16.c.o
[ 89%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_subMagsF32.c.o
[ 89%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/softfloat_state.c.o
[ 89%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_extF80.c.o
[ 89%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/s_subMagsF64.c.o
[ 90%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_extF80M.c.o
[ 90%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_f128.c.o
[ 90%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_f16.c.o
[ 90%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_f128M.c.o
[ 92%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_f32.c.o
[ 92%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui32_to_f64.c.o
[ 92%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_extF80.c.o
[ 92%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_extF80M.c.o
[ 93%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_f128.c.o
[ 93%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_f128M.c.o
[ 93%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_f16.c.o
[ 93%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_f64.c.o
[ 93%] Building C object dependencies/softfloat/CMakeFiles/softfloat.dir/berkeley-softfloat-3/source/ui64_to_f32.c.o
[ 94%] Linking C static library libsoftfloat.a
[ 94%] Built target softfloat
[ 94%] Built target generated_model_rv64d
[ 94%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/riscv_prelude.cpp.o
[ 94%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/riscv_platform_impl.cpp.o
[ 96%] Building C object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/__/riscv_model_rv64d.c.o
[ 96%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/riscv_callbacks.cpp.o
[ 97%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/rvfi_dii.cpp.o
[ 97%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/riscv_platform.cpp.o
[ 97%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/riscv_sim.cpp.o
[ 97%] Building C object c_emulator/CMakeFiles/riscv_sim_rv64d.dir/riscv_softfloat.c.o
$HOME/sail-riscv/build/riscv_model_rv64d.c: In function ‘zwithin_htif_writable’:
$HOME/sail-riscv/build/riscv_model_rv64d.c:55805:15: error: too many arguments to function ‘plat_enable_htif’; expected 0, have 1
55805 |   zgaz31811 = plat_enable_htif(UNIT);
      |               ^~~~~~~~~~~~~~~~
In file included from $HOME/sail-riscv/build/riscv_model_rv64d.c:6:
$HOME/sail-riscv/c_emulator/riscv_platform.h:19:6: note: declared here
   19 | bool plat_enable_htif();
      |      ^~~~~~~~~~~~~~~~
$HOME/sail-riscv/build/riscv_model_rv64d.c: In function ‘zwithin_htif_readable’:
$HOME/sail-riscv/build/riscv_model_rv64d.c:55882:15: error: too many arguments to function ‘plat_enable_htif’; expected 0, have 1
55882 |   zgaz31817 = plat_enable_htif(UNIT);
      |               ^~~~~~~~~~~~~~~~
$HOME/sail-riscv/c_emulator/riscv_platform.h:19:6: note: declared here
   19 | bool plat_enable_htif();
      |      ^~~~~~~~~~~~~~~~
[ 97%] Built target generated_model_rv32d
[ 97%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/riscv_platform_impl.cpp.o
[ 98%] Building C object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/__/riscv_model_rv32d.c.o
[ 98%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/riscv_prelude.cpp.o
[ 98%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/riscv_callbacks.cpp.o
[ 98%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/riscv_platform.cpp.o
[ 98%] Building C object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/riscv_softfloat.c.o
[100%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/rvfi_dii.cpp.o
[100%] Building CXX object c_emulator/CMakeFiles/riscv_sim_rv32d.dir/riscv_sim.cpp.o
$HOME/sail-riscv/build/riscv_model_rv32d.c: In function ‘zwithin_htif_writable’:
$HOME/sail-riscv/build/riscv_model_rv32d.c:55731:15: error: too many arguments to function ‘plat_enable_htif’; expected 0, have 1
55731 |   zgaz31760 = plat_enable_htif(UNIT);
      |               ^~~~~~~~~~~~~~~~
In file included from /$HOME/sail-riscv/build/riscv_model_rv32d.c:6:
$HOME/sail-riscv/c_emulator/riscv_platform.h:19:6: note: declared here
   19 | bool plat_enable_htif();
      |      ^~~~~~~~~~~~~~~~
$HOME/sail-riscv/build/riscv_model_rv32d.c: In function ‘zwithin_htif_readable’:
$HOME/sail-riscv/build/riscv_model_rv32d.c:55808:15: error: too many arguments to function ‘plat_enable_htif’; expected 0, have 1
55808 |   zgaz31766 = plat_enable_htif(UNIT);
      |               ^~~~~~~~~~~~~~~~
$HOME/sail-riscv/c_emulator/riscv_platform.h:19:6: note: declared here
   19 | bool plat_enable_htif();
      |      ^~~~~~~~~~~~~~~~
make[2]: *** [c_emulator/CMakeFiles/riscv_sim_rv64d.dir/build.make:79: c_emulator/CMakeFiles/riscv_sim_rv64d.dir/__/riscv_model_rv64d.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1970: c_emulator/CMakeFiles/riscv_sim_rv64d.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [c_emulator/CMakeFiles/riscv_sim_rv32d.dir/build.make:79: c_emulator/CMakeFiles/riscv_sim_rv32d.dir/__/riscv_model_rv32d.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1830: c_emulator/CMakeFiles/riscv_sim_rv32d.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

--- a/c_emulator/riscv_platform.cpp
+++ b/c_emulator/riscv_platform.cpp
@@ -66,7 +66,7 @@ unit plat_term_write(mach_bits s)
   return UNIT;
 }
 
-bool plat_enable_htif()
+bool plat_enable_htif(unit)
 {
   return rv_enable_htif;
 }

--- a/c_emulator/riscv_platform.h
+++ b/c_emulator/riscv_platform.h
@@ -16,7 +16,7 @@ unit cancel_reservation(unit);
 unit plat_term_write(mach_bits);
 
 mach_bits plat_htif_tohost(unit);
-bool plat_enable_htif();
+bool plat_enable_htif(unit);
 
 bool sys_enable_experimental_extensions(unit);
 

